scripts/travis/horti-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+# For more information on how to use this docker compose file see our docs page:
+#   https://docs.communityhealthtoolkit.org/apps/guides/hosting/self-hosting/
+
+services:
+  couch:
+    image: couchdb:2.3.1
```

Comment:
What's the advantage in adding CouchDB inside the horti container?

Reply:
We already use couch in a container. This just puts it all in one place and starts it all together with `docker-compose up` instead of having to call. 

```
    - name: Couch Start
      run: ./scripts/travis/couch-start
   ```

Reply:
Sorry that didn't answer your question. They are still separate containers just being launched in one place.